src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] Please work on the division of a number with zero let us say `9/0` it should return `error` in the screen but currently its returning
![shot](https://user-images.githubusercontent.com/44745855/133408597-9c70d3b1-5392-4947-a5cb-81b1aac8301d.PNG)
please work on that